weave/mongo_loader.py

```diff
@@ -19,7 +19,6 @@
 from .basket import Basket
 from .config import get_mongo_db
 
-
```

Comment:
There should be two blank lines here
https://pep8.org/#blank-lines

Reply:
Per the pep8 link "Surround top-level function and class definitions with two blank lines.", I think the blank line was supposed to be added before the class, not between the imports

Reply:
Line 22 *is* before the class, not between the imports. I'm confused what you're saying here.

Reply:
I'm saying Line 22 should be added back in, the newest commit after this comment added two blank lines between 19 and 20

Reply:
I agree with your comment, I should've commented on the new commit instead of replying to your message, I was just trying to emphasize to the developer that the comment was not resolved yet in my opinion

Reply:
fixed